kubestellar-logs/kubestellar-where-resolver-log.txt

```diff
@@ -0,0 +1,281 @@
+I0814 23:02:17.436372  490050 where-resolver-cmd.go:206] "Found APIExport view" exportName="edge.kubestellar.io" serverURL="https://10.0.2.15:6443/services/apiexport/310rkkl1jtooufgo/edge.kubestellar.io"
```

Comment:
This and the other logs were unintentional inclusions, right?

Reply:
@MikeSpreitzer yes indeed. It wasn't clear to me what should or shouldn't be included after running tests.